src/components/side_panel/chart/building_blocks/axis_design/axis_design_editor.xml

```diff
@@ -16,5 +16,60 @@
       style="axisTitleStyle"
       defaultStyle="{align: 'center', color: '', fontSize: defaultFontSize}"
     />
+    <Section
+      t-if="!isCategoricalAxis"
+      class="'pt-0 o-axis-bounds-section'"
+      title.translate="Axis range">
+      <div class="d-flex">
+        <div class="w-50">
+          <span class="o-section-subtitle my-0">Minimum</span>
+          <input
```

Comment:
After https://github.com/odoo/o-spreadsheet/pull/7023 we will  avoid using simple inputs, use the new components instead :)